modules/rubiconBidAdapter.js

```diff
@@ -528,6 +528,13 @@ export const spec = {
     if (bidRequest?.ortb2Imp?.ext?.ae) {
       data['o_ae'] = 1;
     }
+
+    const topics = getCarbonTopics(bidderRequest);
+    if (topics) {
+      Object.keys(topics).forEach(field => {
+        data[field] = topics[field];
+      });
+    }
```

Comment:
Not really a huge performance increase but no need to have two loops when we can just add those keys to the data object during the initial getCarbonTopics loop.

And maybe we can generalize it to not say carbon and just "Get wanted segtax" or topics or whatever. And it just takes in list of taxonomy numbers?

So if we need to add another one we just update the list or something. - guess you can just update the if statement also so no worries.